src/Panel/Panel.php

```diff
@@ -883,12 +1055,15 @@ public static function view(array $view = [], ?array $area = null): array
         $view = array_replace_recursive($area ?? [], $view);
 
         $view['breadcrumb'] = $view['breadcrumb'] ?? [];
+        $view['code']       = $view['code'] ?? 200;
         $view['path']       = Str::after($kirby->path(), '/');
+        $view['timestamp']  = time();
```

Comment:
Is the granularity of one second enough? We could use:

```suggestion
        $view['timestamp']  = (int)(microtime(true) * 1000);
```

for milliseconds.